util/a2e/atoe.c

```diff
@@ -1327,6 +1327,53 @@ atoe_putchar(int ch)
 	return putchar((int)a2e_tab[ch]);
 }
 
+/**************************************************************************
```

Comment:
Rather than reordering, suggest just adding declarations for those functions which are referenced before they are defined (we already have some such declarations near line 161).